README.md

```diff
@@ -17,7 +17,7 @@ This package contains the core features of OpenFisca, which are meant to be used
 
 OpenFisca runs on Python 3.7. More recent versions should work, but are not tested.
 
-OpenFisca also relies strongly on NumPy. Last four minor versions should work, but only latest/stable is tested.
+OpenFisca also relies strongly on NumPy. Last four minor versions should work, but only upper and lower bound versions are tested.
```

Comment:
Where do these “last four minor” come from? 🤔

```suggestion
OpenFisca also relies strongly on NumPy. Only upper and lower bound versions are tested.
```

Reply:
To assure compatibility for @openfisca/france-contrib-ipp and @openfisca/france-contrib-leximpact.

But it is still arbitrary: whatever number or road we choose, I think it should be properly documented as it is a contract we assume with our users.

Reply:
Based on the discussion in #1012, I understand that version upgrades are costly based more on time periods when they happen than on the version number difference, so I am not sure a “4 last versions” policy makes any specific sense. But this PR might not be the place to discuss it. It's probably worth having a discussion and opening up an RFC on that topic.

Reply:
My suggestion for rephrasing still stands in the current context 🙂